### ISSUE TITLE
Renamed fields behaves weird when null

### ISSUE THREAD

**Issue by JSteunou** [sequelize/sequelize#2880]
I have this in my models

```
        createdAt: {
            type: DataTypes.DATE,
            field: 'created_at'
        },
        updatedAt: {
            type: DataTypes.DATE,
            field: 'updated_at'
        },
        deletedAt: {
            type: DataTypes.DATE,
            field: 'deleted_at'
        }
```

And I have this case when I create a resource

```
updatedAt: Fri Jan 09 2015 14:06:36 GMT+0100 (CET),
createdAt: Fri Jan 09 2015 14:06:36 GMT+0100 (CET),
deleted_at: undefined
```

All fields are well camelCased except deletedAt which remain like column name. Maybe because it's null?

**Comment by mickhansen** [comment]
Appears there are some issues with `timestamps` and field in general.

**Comment by JSteunou** [comment]
Yep. I revert my use of camelCase for now :/

**Comment by mickhansen** [comment]
Likely fixed in https://github.com/sequelize/sequelize/commit/b61b3e83540ac35d3155631e52e359e052554302

**Comment by JSteunou** [comment]
Still present in rc7

```
db.User.create(user).then(function(user) {
  console.log('===== USER', user.get());
});
```

```
Executing (default): INSERT INTO `user` (`id`,`created_at`,`updated_at`,`deleted_at`,`email`,`password`,`salt`,`uuid`) VALUES (DEFAULT,'2015-01-12 09:24:38','2015-01-12 09:24:38',NULL,'jean.dupont@gmail.com','8696304f3c53e2597a09686d01d145597d60775a','288f164bffd27835db1b0ee55da3438d',X'7d80d078f34c4ed9b1ff1cf65b99aee0');
===== USER {
  id: 1,
  email: 'jean.dupont@gmail.com',
  updatedAt: Mon Jan 12 2015 10:24:38 GMT+0100 (CET),
  createdAt: Mon Jan 12 2015 10:24:38 GMT+0100 (CET),
  deleted_at: undefined }
```

**Comment by mickhansen** [comment]
Fix is not on rc7, its on master.

**Comment by mickhansen** [comment]
https://github.com/sequelize/sequelize/commit/b61b3e83540ac35d3155631e52e359e052554302 is not tagged to any release as you can see :)

**Comment by JSteunou** [comment]
Tested against master this time, check & double check, still same issue & output.

**Comment by JSteunou** [comment]
Note it's not a big issue though.

**Comment by mickhansen** [comment]
Hmm alright, i'll have to give it another look.

**Comment by JSteunou** [comment]
Fixed with same commit as #2879 (so in rc8) for me.
